src/js/index.js

```diff
@@ -7,3 +7,24 @@ navigation()
 bookmark()
 formular()
 quizCardAnswer()
+
+// IDEA: randomizer button for random question
+
+const quizCard = document.querySelector('[data-js=quiz-card]')
+const buttonRandom = document.querySelector('[data-js=randomButton]')
+const number = quizCard.length
```

Comment:
You need the length of the quiz card array
```suggestion
const number = quizCards.length
```